.github/workflows/publish_docker_image.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish Docker image
+on:
+  push:
+    branches:
+      - main
+jobs:
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to GitHub Packages
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          registry: docker.pkg.github.com
+          repository: open-privacy/opv/main
+          tag_with_ref: true
```

Comment:
I wonder if `latest` tag is also there or not?

Reply:
@zhouzhuojie updated

Reply:
Likely should need another workflow to push and tag releases, but that's a different story.